tchapgouv/tchap-android
# Hotfix/tchap v2.4.4

## Type of change


## Content

Change the push handler for FirebaseMessagingService.
Fix the issue on Android 13 at first start

## Motivation and context

Many customers complained about not receiving the push notification on their devices.
We decided to cherrypick this fix from element

## Tests



- Create a room between two users
- Put one of the phone in lock mode
- Send a message from the other user's device
- Notification should appear on lock screen and device should emit sound / vibrate

## Tested devices

- OS version(s): Android 13

## Checklist

- [ ] Accessibility has been taken into account. See https://github.com/tchapgouv/tchap-android-v2/blob/develop/CONTRIBUTING.md#accessibility
- [ ] Pull request updates [TCHAP_CHANGES.md](https://github.com/tchapgouv/tchap-android-v2/blob/develop/TCHAP_CHANGES.md)
- [ ] Pull request includes a new file under ./changelog.d. See https://github.com/tchapgouv/tchap-android-v2/blob/develop/CONTRIBUTING.md#changelog
- [ ] If you have modified the screen flow, or added new screens to the application, you have updated the test [UiAllScreensSanityTest.allScreensTest()](https://github.com/tchapgouv/tchap-android-v2/blob/develop/vector/src/androidTest/java/im/vector/app/ui/UiAllScreensSanityTest.kt)
